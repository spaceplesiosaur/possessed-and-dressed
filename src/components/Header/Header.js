import React from 'react';
import './Header.scss';

export const Header = () => {
  return (
    <header className="app-header">Posessed & Dressed</header>
  )
}

export default Header;
