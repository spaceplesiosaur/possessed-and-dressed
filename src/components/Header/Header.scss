$dark-text-color: #FA1E88;
$bright-header-color: #282c34;

.app-header {
  background-color: $dark-text-color;
  display: flex;
  flex-direction: column;
  align-items: center;
  justify-content: flex-start;
  color:$bright-header-color;
  font-size: 5em;
  font-style: italic;
  padding: 3rem;
}

.app-header:link {
  text-decoration: none;
}
