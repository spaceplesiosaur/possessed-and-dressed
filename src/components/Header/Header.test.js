import React from 'react';
import Header from './Header'
import { shallow } from 'enzyme'

describe('Header', () => {
  it('should render with correct data', () => {
    const wrapper = shallow(
      <Header />
    )

    expect(wrapper).toMatchSnapshot()
  })
})
