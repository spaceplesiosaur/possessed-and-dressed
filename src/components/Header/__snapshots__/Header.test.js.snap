// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header should render with correct data 1`] = `
<Link
  to="/"
>
  <header
    className="app-header"
  >
    Posessed & Dressed
  </header>
</Link>
`;
